ChegeBryan/black-bandana
# #163261740 User Registration API

#### What does this PR do?

User basic registration

#### Description of Task to be completed?

- Write test data for user registration on tests 
- Write tests for user POST API
- Create routes for creating a user.

#### How should this be manually tested?
After cloning the repo to your local machine - check [README.md](README.md) for instructions. checkout to the branch `ft-register-user-api-163261740` Using Postman open a `POST` request with this header
Key Content-Type: application/json
Enter the following in the request body in json 
```
{
    "user_name": "test username",
    "user_email": "user@emai.com",
    "password": "justanoter23"
}
```

#### What are the relevant pivotal tracker stories?
[#163261740](https://www.pivotaltracker.com/story/show/163261740)

#### Postman Documentation
https://documenter.getpostman.com/view/4869820/RzthQWgJ

#### Screenshots (if appropriate)

#####  `POST /users`

![screenshot from 2019-01-24 16-39-11](https://user-images.githubusercontent.com/40359451/51682175-cdb3b300-1ff7-11e9-8e79-4fc3f62ab119.png)
